controllers/userController.js

```diff
@@ -19,23 +25,35 @@ export default class UserController {
   }
 
   // Create New User.
-  createUser(req) {
+  async createUser(req) {
     // Get POST params
     const {
       email, phoneNo, fullName, password,
     } = req.body;
 
+    const hashedPassword = bcrypt.hashSync(password, 10);
+
     const newUser = new User(email, fullName,
-      phoneNo, password);
+      phoneNo, hashedPassword, new Date(), roleId);
 
-    const userFound = this.mapUserList.get(email);
-    const status = (userFound === undefined) ? 201 : 409;
+    const userFound = await db.query(userquery.queryUser(email));
+    const status = (userFound.rowCount === 1) ? 409 : 201;
 
     // Populate List in Memory if object is not empty
     if (!(validate(newUser)) && status === 201) {
-      this.mapUserList.set(newUser.email, newUser);
-      this.response = new Response('ok', status, 'New user Created', newUser);
+      const success = await db.query(
+        userquery.createUser(
+          newUser.email, newUser.fullName,
+          newUser.phoneNo, newUser.password,
+          newUser.dateCreated, newUser.roleId,
+        ),
+      );
+      if (success.rowCount === 1) {
+        newUser.password = '';
```

Comment:
I think there's a `delete` method that can be used to delete keys from an object. You could use that so that the `password` key doesn't appear in the response

Reply:
Will implement this now